### DISCUSSION TITLE
Unable to save workflow

### DISCUSSION THREAD

**Discussion by komalphadatare** [StackStorm/st2#5614] [Q&A]
Hi,
Stackstorm Version : 3.6.0
OS : Centos7
While Saving workflow we are getting below error
×Error saving workflow.: Additional properties are not allowed ('_name' was unexpected)

I see that above error is being fixed in 3.7 version as mentioned below link
https://github.com/StackStorm/st2web/pull/961 

In "https://docs.stackstorm.com/latest/install/index.html" link provides installation steps for 3.6. Can you please share the installation documentation link to install 3.7.

Regards,
Komal P

**Comment by amanda11** [comment]
StackStorm 3.7 hasn't yet been released yet, we are expecting to release it this month (just a few little bits left to finish).

The fix is available in the 3.7dev repositories, and so you can install with the "unstable" option to get the in-development release.  Most of our installers support using the "unstable" option to install the in-dev release, but note that as its in the in-development release the versio of that can change daily.

Alternatively in a few weeks StackStorm 3.7 will be released, which will include this fix.

**Comment by amanda11** [comment]
And to give some context on the problem, the Save only happens if you have previously hit the run button from the edit workflow page. A workaround with 3.6 is to have two tabs, and to run the workflow from the Actions tab, and use your edit workflow tab just for editing and saving workflow.
